### PR TITLE
add elizaServiceClient.flush()

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner

### CODE REVIEW

**Comment by andrewconner** [comment]
Word

File: kifi-backend/modules/common/app/com/keepit/eliza/ElizaServiceClient.scala [hunk 1]
```diff
@@ -190,6 +186,11 @@ class ElizaServiceClientImpl @Inject() (
     broadcast(Eliza.internal.sendToAllUsers, data)
   }
 
+  def flush(userId: Id[User]): Future[Unit] = {
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/eliza/ElizaServiceClient.scala:189]: Thanks for returning `Future[Unit]`

**Comment by andrewconner** [comment]
Merging because I want this

File: kifi-backend/modules/common/app/com/keepit/eliza/ElizaServiceClient.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/common/app/com/keepit/eliza/ElizaServiceClient.scala:189]: That's just to let us know when the future completes right?

**Pull request merged**